Home.md

```diff
@@ -17,3 +17,73 @@ Use the [`#WindowsCommunityToolkit`](https://twitter.com/search?q=%23WindowsComm
 - Windows Blogs - Release Overviews - [Search for Windows Community Toolkit](https://blogs.windows.com/?s=Windows+Community+Toolkit)
 - Dev Blogs - [Other Articles](https://devblogs.microsoft.com/pax-windows/)
 - We may sometimes add to the [.NET Foundation Newsletter](https://github.com/dotnet-foundation/website/blob/master/input/blog/posts/_current-newsletter-draft.md) too!
+
+## Projects that are currently using Windows Community Toolkit...
+
+<h1 align="center">Fiona</h1>
+
+<table>
+  <tr>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona1.png height=350></td>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona2.png height=350></td>
+  </tr>
+  <tr>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona4.png height=350></td>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona3.png height=350></td>
+  </tr>
+</table>
+
+>Creator: [PaoloM](https://github.com/PaoloM)
+
+<p align="left">Fiona was started using the WCT Template, and it's using a bunch of the helpers and services provided with that. Plus, as mentioned, the toolkit MVVM.</p>
```

Comment:
```suggestion
Fiona was started using the WCT Template ([from Windows Template Studio](https://aka.ms/wts)), and it's using a bunch of the helpers and services provided with that. Plus, as mentioned, the toolkit MVVM.
```
We can probably switch the HTML to Markdown here for the headers and things to keep it a bit more simple.